src/components/LoadingSkeletons/BarChartLoadingSkeleton.tsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import LoadingMessage from '../LoadingMessage/LoadingMessage';
+import { cxBackgroundGray } from './LoadingSkeletonsSvgUtil';
+import LoadingSkeleton, {
+	ILoadingSkeletonProps,
+	IStandardSkeleton,
+} from './LoadingSkeleton';
+
+export const BarChartSkeleton = (
+	props: IStandardSkeleton
+): React.ReactElement => {
+	const { width, height, className } = { ...props };
+
+	return (
+		<div
+			data-test-id='loadingSkeleton-BarChartSkeleton'
+			className={cxBackgroundGray('&', className)}
+		>
+			<svg
+				data-test-id='loadingSkeleton-BarChartSkeleton-svg'
+				width={width}
+				height={height}
+				version='1.1'
+			>
+				<g
+					id='Details'
+					stroke='none'
+					strokeWidth='1'
+					fill='none'
+					fillRule='evenodd'
+				>
+					<g
+						id='Analytics-Tab_Skeleton-DM'
```

Comment:
Can this `id` be specific related to this component? I feel this might be clashing with the consumer.

Reply:
Done